spec/cobrato/resources/regress_cnab_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.